config.gemspec

```diff
@@ -37,7 +37,7 @@ Please consider donating to our open collective to help us maintain this project
   s.add_development_dependency 'rspec', '~> 3.7', '>= 3.7.0'
 
   # Default RSpec run will test against latest Rails app
-  unless ENV['APPRAISAL_INITIALIZED']
+  unless ENV['APPRAISAL_INITIALIZED'] || ENV['CI']
```

Comment:
I think `ENV['APPRAISAL_INITIALIZED']` should be enough, don't you think?

Reply:
/shrug I thought so, but it was failing on travis before this change